shared/src/main/java/uk/gov/di/authentication/shared/services/CodeStorageService.java

```diff
@@ -50,6 +52,16 @@ public int getIncorrectMfaCodeAttemptsCount(String email) {
                                         + HashHelper.hashSha256String(email)));
         return count.map(Integer::parseInt).orElse(0);
     }
+
+    public int getIncorrectMfaCodeAttemptsCount(String email, MFAMethodType mfaMethodType) {
+        Optional<String> count =
+                Optional.ofNullable(
+                        redisConnectionService.getValue(
+                                MULTIPLE_INCORRECT_MFA_CODES_KEY_PREFIX
+                                        + mfaMethodType.getValue()
+                                        + HashHelper.hashSha256String(email)));
+        return count.map(Integer::parseInt).orElse(0);
+    }
     // TODO: remove this transitional method when cache reflects MFA-type-specific prefixing i.e.
```

Comment:
Same as above. Can we also remove it for the `deleteIncorrectMfaCodeAttemptsCount`

Reply:
Sure :+1: